microsoft/testsuites/core/provisioning.py

```diff
@@ -31,7 +31,50 @@ class Provisioning(TestSuite):
 
     @TestCaseMetadata(
         description="""
-        This case verifies whether a node is operating normally.
+        This case runs smoke test on a node with standard disk.
+        """,
+        priority=0,
+        requirement=simple_requirement(
+            environment_status=EnvironmentStatus.Deployed,
+            supported_features=[SerialConsole],
+        ),
+    )
+    def smoke_test(self, case_name: str, node: RemoteNode) -> None:
+        self._smoke_test(case_name, node)
+
+    @TestCaseMetadata(
+        description="""
+        This case runs smoke test on a node provisioned with emphemeral disk.
+        """,
+        priority=1,
+        requirement=simple_requirement(
+            environment_status=EnvironmentStatus.Deployed,
+            supported_features=[SerialConsole, DiskEphemeral],
+        ),
+    )
+    def verify_deployment_provision_ephemeral_managed_disk(
+        self, case_name: str, node: RemoteNode
+    ) -> None:
+        self._smoke_test(case_name, node)
+
+    @TestCaseMetadata(
+        description="""
+        This case runs smoke test on a node provisioned with premium disk.
+        """,
+        priority=1,
+        requirement=simple_requirement(
+            environment_status=EnvironmentStatus.Deployed,
+            supported_features=[SerialConsole, DiskPremiumLRS],
+        ),
+    )
+    def verify_deployment_provision_premium_disk(
+        self, case_name: str, node: RemoteNode
+    ) -> None:
+        self._smoke_test(case_name, node)
+
+    def _smoke_test(self, case_name: str, node: RemoteNode) -> None:
+        """
+        This function verifies whether a node is operating normally.
```

Comment:
Those content should be put on `smoke_test` description. It's a part of test spec, so it can be included in future documentation. For other test cases, they need explain that the progress is the same as `smoke_test`.